docs/issue-labels.rst

```diff
@@ -7,93 +7,89 @@ what they stand for.
     project leader clarifies the way in which the issue should be approached.
 
 *Needed: documentation*
     If an issue involves creating or refining documentation, this label will be
     assigned.
 
 *Needed: more information*
-    This label indicates that the issue has not enough information in order to
-    decide on how to go forward. See the documentation about our :ref:`triage
-    process <triage-not-enough-information>` for more information.
+    This label indicates that a reply with more information is required from the
+    bug reporter. If no response is given by the reporter, the issue is
+    considered invalid after 7 days and will be closed. See the documentation
+    about our :ref:`triage process <triage-not-enough-information>` for more
+    information.
 
 *Needed: patch*
     This label indicates that a patch is required in order to resolve the
-    ticket. A fix should be proposed via a pull request on GitHub.
+    issue. A fix should be proposed via a pull request on GitHub.
 
 *Needed: tests*
     This label indicates that a better test coverage is required to resolve
-    the ticket. New tests should be proposed via a pull request on GitHub.
+    the issue. New tests should be proposed via a pull request on GitHub.
 
 *Needed: replication*
     This label indicates that a bug has been reported, but has not been
     successfully replicated by another user or contributor yet.
 
 *Operations*
-    Tickets that require changes in the server infrastructure.
-
-*PR: ready for review*
-    Pull Requests that are considered complete. A review by at least one core
-    developer is required prior to merging it.
+    Issues that require changes in the server infrastructure.
 
 *PR: work in progress*
     Pull Requests that are not complete yet. A final review is not possible
     yet, but every Pull Request is open for discussion.
 
 *Sprintable*
-    Sprintable are all tickets that have the right amount of scope to be
+    Sprintable are all issues that have the right amount of scope to be
     handled during a sprint. They are very focused and encapsulated.
 
 *Status: blocked*
-    The ticket cannot be resolved until some other ticket has been closed.
-    See the ticket's log for which ticket is blocking this ticket.
-
-*Status: duplicate*
-    See the ticket's log to find the original ticket that this one is a
-    duplicate of.
-
-*Status: invalid*
-    A ticket is invalid if the reported issue cannot be reproduced.
+    The issue cannot be resolved until some other issue has been closed.
+    See the issue's log for which issue is blocking this issue.
 
-*Status: rejected*
-    A ticket gets rejected if the proposed enhancement is not in line with the
-    overall goals of readthedocs.org.
+*Status: stale*
+    A issue is stale if it there has been no activity on it for 90 days. Once a
+    issue is determined to be stale, it will be closed in 7 days unless there
+    is activity on the issue.
```

Comment:
Not sure the real value here. Do we imagine a bot adding a `stale` tag, and then that would magically get someone to update the ticket?

Reply:
Yeah, this is strictly for probot automation in #4111